docs/deployment/scaling.md

```diff
@@ -67,3 +67,131 @@ You must consider that the targets for CPU and Memory use the service replicas l
+
+
+To autoscale based on Datadog metrics you must first have Datadog and the Datadog Cluster Agent installed. If you used the `datadog-agent-all-features.yaml` when [configuring Datadog](/integrations/monitoring/datadog/) then you will already have the Datadog Cluster Agent installed.
+
+You can check your cluster's pods to see what is installed:
+* If you installed into a custom namespace you will have to add the appropriate `-n NAMESPACE_NAME` option to the end of the command
+```html 
+    $ kubectl get pods                                                          
+    NAME                                    READY   STATUS    RESTARTS   AGE 
+    datadog-bjw2m                           5/5     Running   0          16m 
+    datadog-cluster-agent-b5fd4b7f5-tmkql   1/1     Running   0          16m 
+    datadog-j9c9t                           5/5     Running   0          15m 
+    datadog-pnrln                           5/5     Running   0          16m 
+    datadog-vdzc5                           5/5     Running   0          16m 
+``` 
+
+If your installation does not have an active datadog-cluster-agent pod you can install the Datadog Cluster Agent by following the instructions from the [Datadog Documentation](https://docs.datadoghq.com/containers/cluster_agent/setup/?tab=daemonset). 
+
+
+Once the Datadog Cluster Agent is setup follow this [Datadog Documentation](https://docs.datadoghq.com/containers/guide/cluster_agent_autoscaling_metrics/?tab=daemonset#register-the-external-metrics-provider-service) to configure your Datadog Cluster Agent with an external metrics provider service.
+
+
+Next to setup DatadogMetric follow these steps from the [Datadog Documentation](https://docs.datadoghq.com/containers/guide/cluster_agent_autoscaling_metrics/?tab=daemonset#datadog-cluster-agent).
+
+You can now create scaling manifests based on Datadog's provided template:
+```html 
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogMetric
+metadata:
+  name: <your_datadogmetric_name>
+spec:
+  query: <your_custom_query>
+``` 
+You can deploy these metrics with `kubectl apply -f your-custom-metric-file.yaml`
+
+You will finally then need to deploy a service for your apps to push metrics to Datadog.  You can use this example or write your own based on your own custom configurations:
+```html 
+$ cat dd-agent-service.yaml
+apiVersion: v1 
+kind: Service 
+metadata: 
+  name: dd-agent 
+spec: 
+  selector: 
+    app: datadog 
+  ports: 
+    - protocol: UDP 
+      port: 8125 
+      targetPort: 8125 
+``` 
+Deploy the service with `kubectl apply -f dd-agent-service.yaml`
+
+
+In this example we will start with a simple [nodejs app](https://github.com/convox-examples/nodejs) to demonstrate how to setup scaling with this method. 
```

Comment:
`https://github.com/convox-examples/nodejs` to `https://github.com/convox-examples/nodejs-dd-autoscale`?